packages/react-router5/modules/hocs/routeNode.tsx

```diff
@@ -4,7 +4,7 @@ import RouteNode from '../render/RouteNode'
 
 function routeNode<P>(nodeName: string) {
     return function(BaseComponent: ComponentType<P & RouteContext>): SFC<P> {
-        function RouteNodeWrapper(props) {
+        function RouteNodeWrapper(props: P) {
```

Comment:
I dunno why this is needed but vscode complained. It should have been infered from SFC<P>